usaspending_api/download/filestreaming/file_b_custom_account_download_export.sql

```diff
@@ -0,0 +1,178 @@
+-- Connections are pooled which means it's possible for temporary tables to survive beyond what we
```

Comment:
Since it's a temp table I assumed there would be no bleed-over between download jobs.

Based on this comment, should the temporary table name be unique?

Reply:
The entire SQL statement is being run in a single transaction so there (theoretically) can't be any bleed-over.  This is just an (almost) zero cost safety precaution.

Reply:
I need to think on this some more.  I need to reproduce my findings from where I saw temporary tables live longer than I expected.  Now I'm wondering if that was at the cursor level or the connection level.  I may not have made that distinction when I discovered this a few months ago.  If it's at the cursor level, this is a non-issue.  If it's at the connection level, we may have a blocking issue that unique names should alleviate.

Reply:
Bottom line is, in Django, temp tables live at the connection level and can live longer than a "session", however, in our implementation we're spawning a `psql` subprocess to run these queries.  Subproceses obtain their own connection and should therefore not be able to block or otherwise conflict with each other.

Having said that, I think there's value in randomizing the names a bit since I don't know how this code is going to be used in the future.  So in the spirit of forward thinking, I will add some randomness to the names just to "guarantee" (not really guaranteed) there can be no collisions.

It is trivial to implement and will cost nothing at the query/code level.